plugin/src/main/java/com/gtnewhorizons/retrofuturagradle/mcp/MCPTasks.java

```diff
@@ -288,12 +298,29 @@ public MCPTasks(Project project, MinecraftExtension mcExt, MinecraftTasks mcTask
                 });
         postprocessedMcChain.addTask(taskApplySourceAccessTransformers);
 
+        interfaceInjectedJarLocation = FileUtils.getFile(buildDir, RFG_DIR, "srg_patched_injected_minecraft-sources.jar");
+        taskInjectInterfaces = project.getTasks()
```

Comment:
Can you merge this into the JST-using AT task to just do one pass over the jar instead of two? Can call it jstTransform or somthing similar